tests/test_ts.py

```diff
@@ -204,15 +184,38 @@ def test_custom_sample_ids(self, simple_ts):
 
     def test_sample_id_length_mismatch(self, simple_ts):
         ts_path, _ = simple_ts
-        ind_nodes = np.array([[0, 1], [2, 3]])
-        # Wrong number of sample IDs
         with pytest.raises(ValueError, match="Length of sample_ids.*does not match"):
-            ts.TskitFormat(ts_path, ind_nodes, sample_ids=["only_one_id"])
+            ts.TskitFormat(ts_path, sample_ids=["only_one_id"])
+
+    def test_no_individuals_ts(self, tmp_path):
+        tables = tskit.TableCollection(sequence_length=100)
+        # tables.nodes.add_row(flags=tskit.NODE_IS_SAMPLE, time=0)
```

Comment:
Delete comments...

Reply:
Gees, sorry! Can I blame the fact that syntax highlighting is broken in my editor?